docs/03-proof_submission_interface/03-polygon_cdk_proof_submission.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Tutorial: How to submit a Polygon CDK proof
```

Comment:
For some reason there is an issue with this line.
Could be related to the extra colon `:` or some extra newline character.
<img width="611" alt="image" src="https://github.com/HorizenLabs/zkverify-docs/assets/88201146/8411d1eb-f421-4124-83b3-fd41dd001c66">

Reply:
```

08:57:18.797 | [ERROR] Error: Unable to build website for locale en.
-- | --
08:57:18.797 | at tryToBuildLocale (/opt/buildhome/repo/node_modules/@docusaurus/core/lib/commands/build.js:53:19)
08:57:18.797 | at async /opt/buildhome/repo/node_modules/@docusaurus/core/lib/commands/build.js:64:9
08:57:18.797 | at async mapAsyncSequential (/opt/buildhome/repo/node_modules/@docusaurus/utils/lib/jsUtils.js:20:24)
08:57:18.797 | at async Command.build (/opt/buildhome/repo/node_modules/@docusaurus/core/lib/commands/build.js:62:5) {
08:57:18.797 | [cause]: Error: Can't process doc metadata for doc at path path=/opt/buildhome/repo/docs/03-proof_submission_interface/03-polygon_cdk_proof_submission.md in version name=current
08:57:18.797 | at processDocMetadata (/opt/buildhome/repo/node_modules/@docusaurus/plugin-content-docs/lib/docs.js:139:15)
08:57:18.797 | at async Promise.all (index 14)
08:57:18.798 | at async doLoadVersion (/opt/buildhome/repo/node_modules/@docusaurus/plugin-content-docs/lib/index.js:91:34)
08:57:18.798 | ... 11 lines matching cause stack trace ...
08:57:18.798 | at async Command.build (/opt/buildhome/repo/node_modules/@docusaurus/core/lib/commands/build.js:62:5) {
08:57:18.798 | [cause]: YAMLException: incomplete explicit mapping pair; a key node is missed; or followed by a non-tabulated empty line at line 2, column 16:
08:57:18.798 | title: Tutorial: How to submit a Polygon CDK proof
08:57:18.798 | ^
08:57:18.798 | at generateError (/opt/buildhome/repo/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:167:10)
08:57:18.798 | at throwError (/opt/buildhome/repo/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:173:9)
08:57:18.798 | at readBlockMapping (/opt/buildhome/repo/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:1027:9)
08:57:18.799 | at composeNode (/opt/buildhome/repo/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:1359:12)
08:57:18.799 | at readDocument (/opt/buildhome/repo/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:1525:3)
08:57:18.799 | at loadDocuments (/opt/buildhome/repo/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:1588:5)
08:57:18.799 | at load (/opt/buildhome/repo/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:1614:19)
08:57:18.799 | at Object.safeLoad (/opt/buildhome/repo/node_modules/gray-matter/node_modules/js-yaml/lib/js-yaml/loader.js:1637:10)
08:57:18.799 | at module.exports (/opt/buildhome/repo/node_modules/gray-matter/lib/parse.js:12:17)
08:57:18.799 | at parseMatter (/opt/buildhome/repo/node_modules/gray-matter/index.js:109:17)
08:57:18.799 | at matter (/opt/buildhome/repo/node_modules/gray-matter/index.js:50:10)
08:57:18.799 | at parseFileContentFrontMatter (/opt/buildhome/repo/node_modules/@docusaurus/utils/lib/markdownUtils.js:193:57)
08:57:18.800 | at DEFAULT_PARSE_FRONT_MATTER (/opt/buildhome/repo/node_modules/@docusaurus/utils/lib/markdownUtils.js:213:54)
08:57:18.800 | at parseMarkdownFile (/opt/buildhome/repo/node_modules/@docusaurus/utils/lib/markdownUtils.js:269:75)
08:57:18.800 | at doProcessDocMetadata (/opt/buildhome/repo/node_modules/@docusaurus/plugin-content-docs/lib/docs.js:37:108)
08:57:18.800 | at processDocMetadata (/opt/buildhome/repo/node_modules/@docusaurus/plugin-content-docs/lib/docs.js:136:22)
08:57:18.800 | at processVersionDoc (/opt/buildhome/repo/node_modules/@docusaurus/plugin-content-docs/lib/index.js:80:58)
08:57:18.800 | at Array.map (<anonymous>)
08:57:18.800 | at loadVersionDocsBase (/opt/buildhome/repo/node_modules/@docusaurus/plugin-content-docs/lib/index.js:88:45)
08:57:18.800 | at async doLoadVersion (/opt/buildhome/repo/node_modules/@docusaurus/plugin-content-docs/lib/index.js:91:34)
08:57:18.800 | at async loadVersion (/opt/buildhome/repo/node_modules/@docusaurus/plugin-content-docs/lib/index.js:132:28)
08:57:18.801 | at async Promise.all (index 0)
08:57:18.801 | at async Object.loadContent (/opt/buildhome/repo/node_modules/@docusaurus/plugin-content-docs/lib/index.js:140:33)
08:57:18.801 | at async /opt/buildhome/repo/node_modules/@docusaurus/core/lib/server/plugins/plugins.js:38:23
08:57:18.801 | at async Promise.all (index 0)
08:57:18.801 | at async /opt/buildhome/repo/node_modules/@docusaurus/core/lib/server/plugins/plugins.js:139:25
08:57:18.801 | at async loadSite (/opt/buildhome/repo/node_modules/@docusaurus/core/lib/server/site.js:127:45)
08:57:18.801 | at async buildLocale (/opt/buildhome/repo/node_modules/@docusaurus/core/lib/commands/build.js:105:18)
08:57:18.801 | at async tryToBuildLocale (/opt/buildhome/repo/node_modules/@docusaurus/core/lib/commands/build.js:46:13)
08:57:18.801 | at async /opt/buildhome/repo/node_modules/@docusaurus/core/lib/commands/build.js:64:9
08:57:18.801 | at async mapAsyncSequential (/opt/buildhome/repo/node_modules/@docusaurus/utils/lib/jsUtils.js:20:24)
08:57:18.802 | at async Command.build (/opt/buildhome/repo/node_modules/@docusaurus/core/lib/commands/build.js:62:5) {
08:57:18.802 | reason: 'incomplete explicit mapping pair; a key node is missed; or followed by a non-tabulated empty line',
08:57:18.802 | mark: Mark {
08:57:18.802 | name: null,
08:57:18.802 | buffer: '\ntitle: Tutorial: How to submit a Polygon CDK proof\n\x00',
08:57:18.802 | position: 16,
08:57:18.802 | line: 1,
08:57:18.802 | column: 15
08:57:18.802 | }
08:57:18.802 | }
08:57:18.803 | }
08:57:18.803 | }


```